app/Http/Controllers/ApiPhotosController.php

```diff
@@ -239,11 +256,44 @@ public function addTags (AddTagsApiRequest $request)
             'request' => $request->all()
         ]);
 
-        dispatch (new AddTags($request->all(), $user->id));
+        dispatch (new AddTags($user->id, $photo->id, $request->litter));
```

Comment:
I have raised this on mobile too

Reply:
We can't change it to 'tags' immediately because of old app versions that some user might still have.
What we can do is accept both 'litter' and 'tags', and until we're sure that all users use the newer version with 'tags' we keep it this way. I'm implementing this.